products-service/swagger.yaml

```diff
@@ -0,0 +1,74 @@
+openapi: '3.0.2'
+info:
+  title: Bells and Whistles Shop API
+  version: '1.0'
+servers:
+  - url: https://2o4c9nhcai.execute-api.us-west-1.amazonaws.com/dev/
+components:
+  schemas:
+    UUID:
+      type: string
+      pattern: ^[0-9a-f]{8}-[0-9a-f]{4}-[0-5][0-9a-f]{3}-[089ab][0-9a-f]{3}-[0-9a-f]{12}$
```

Comment:
We could just say `format: uuid` instead of pattern and it should work as well

Reply:
I know that Open API doesn't define this format. But I found after your comment:

> However, `format` is an open value, so you can use any formats, even not those defined by the OpenAPI Specification

Of course, `format: uuid` is very discriptive. But I believe it's better to use at least both. Because if we will want try to use this file with codegenerators, it may cause problem, if generator doesn't know `uuid`. And what version of uuid? v2? v4?

It will work only in case, if we already have ecosystem and can strongly answer all questions and get rid of any ambiguity.